releases/protocol/release-specs-v1.9.0-Andromeda.md

```diff
@@ -32,25 +32,44 @@ With this feature, the chain parameters(round duration, consensus group size, mi
 - [#4819](https://github.com/multiversx/mx-chain-go/pull/4819) - Fixes for nodes coordinator with chain parameters
 - [#4927](https://github.com/multiversx/mx-chain-go/pull/4927) - Chain parameters notifier
 
-## 2. Equivalent proofs [#5786](https://github.com/multiversx/mx-chain-go/pull/5786)
+## 2. Fixed ordering in consensus [#6436](https://github.com/multiversx/mx-chain-go/pull/6436)
+
+Each eligible node is now required to sign and the order of signature aggregation was fixed to the order of the eligible nodes decided at epoch start.
```

Comment:
This phrasing is a bit hard to read, for me at least

Reply:
rephrased a bit